packages/e2e-cypress/package.json

```
@@ -41,13 +41,13 @@
   },
   "devDependencies": {
     "@types/lodash": "^4.14.202",
-    "cypress": "^13.6.6",
-    "eslint-plugin-cypress": "^2.15.1",
+    "cypress": "^14.0.1",
+    "eslint-plugin-cypress": "^3.6.0",
     "rimraf": "^5.0.5"
   },
   "peerDependencies": {
-    "cypress": "^12.2.0 || ^13.2.0",
-    "eslint-plugin-cypress": "^2.12.1"
+    "cypress": "^14.0.1",
+    "eslint-plugin-cypress": "^3.6.0"
```

We don't do anything special with the plugin besides installing it. So, we can accept any version really. We could even remove it from peer deps, but let's keep it for now.
```suggestion
    "eslint-plugin-cypress": ">=2.0.0"
```